frontend/src/lib/lemon-ui/Link/Link.tsx

```diff
@@ -43,7 +46,11 @@ const shouldForcePageLoad = (input: any): boolean => {
 }
 
 const isPostHogDomain = (url: string): boolean => {
```

Comment:
I guess this should be `isPostHogCloudDomain` if there is also `isPostHogComDomain`

Reply:
Not quite, the first one is used to determine if it generally is a domain of ours, the other is explicitly the marketting website